README.md

```diff
@@ -179,35 +179,46 @@ f_lower_case_name = Feature(
 
 ### Installing Development Dependencies
 
-1. Install required Python libraries in your Python environment.
+1. Install the required Python libraries.
 
 ```bash
 $ python -m pip install -r python/dev-requirements.txt
 ```
+ 
+2. Start docker engine and pull the required images
 
-2. Make sure you have installed Protocol Buffers compiler 3.17.x in your local
-environment. You may refer to [Protocol Buffers'
-README](https://github.com/protocolbuffers/protobuf#protocol-compiler-installation)
-for installation guidelines. 
+```bash
+$ docker image pull redis:latest
+$ docker image pull confluentinc/cp-kafka:5.4.3
+```
+
+3. Increase open file limit.
+
+```bash
+$ ulimit -n 1024
```

Comment:
Since in #45 we have greatly reduced the Flink environments created during tests, increasing the open file limit might be unnecessary.

Reply:
Due to bug in pyflink, the number of open file descriptor will increase as the number of test classes increase. I tested locally and found that FeatHub python tests will fail if number of open file limit is 256. Setting this limit to 1024 would allow tests to pass.